apps/docs/src/guide/testing/snippets/launching-a-test-node.ts

```diff
@@ -121,7 +121,7 @@ const nodeWithCustomBaseAssetId = await launchTestNode({
     snapshotConfig: {
       chainConfig: {
         consensus_parameters: {
-          V1: {
+          V2: {
```

Comment:
Is this a breaking change?

Reply:
Breaking change for the chain config, not for our SDK consumers. I think.

Reply:
IMO it is not directly a breaking change and we should prioritise getting parity with mainnet as inconsistencies are more dangerous. The core team themselves have upgraded from `v1` -> `v2` over the last 4 months and it was considered non breaking.